package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Shouldn't that just be the https://babeljs.io/docs/plugins/transform-object-rest-spread/ plugin then instead of a stage 2/3 if that is the only thing it needs.